zephyr/wrapper.c

```diff
@@ -166,7 +193,13 @@ void rfree(void *ptr)
 	if (!ptr)
 		return;
 
-	heap_free(&sof_heap, ptr);
+	/* select heap based on address range */
+	if (!is_uncached(ptr)) {
```

Comment:
double negation ;-) quite easy to avoid here

Reply:
@lyakh Yes, I'll fix this.